e2e/applitools.config.js

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// See https://github.com/applitools/eyes.sdk.javascript1/tree/master/packages/eyes-cypress#advanced-configuration
+
+module.exports = {
+    appName: 'Mattermost Webapp UI',
+    accessibilityValidation: {level: 'AA', guidelinesVersion: 'WCAG_2_0'},
+    batchName: 'Webapp - master (dev)',
```

Comment:
Quick question: is batchName here supposed to be a default value? Reason I asked is when I don't set APPLITOOLS_BATCH_NAME, the value of the batch name in the UI reports is `undefined`.

Reply:
Oh, good catch. Yes, that should be the default value. Should be fixed now, PTAL.